frontend/src/i18n/locales/features/login/en-sg.ts

```diff
@@ -12,5 +21,15 @@ export const enSG: Login = {
       login: 'Log in',
       haveAQuestion: 'Have a question?',
     },
+    OTPForm: {
+      signin: 'Sign in',
+      otpRequired: 'OTP is required.',
+      otpLengthCheck: 'Please enter a 6 digit OTP.',
+      otpTypeCheck: 'Only numbers are allowed.',
+      otpFromEmail: 'Enter OTP sent to {email}',
+    },
+    SgidLoginButton: {
+      selectAgencies: 'For <2><0>select agencies</0></2>',
```

Comment:
@KenLSM - I saw your comment about avoiding coupling to DOM structure. would this also apply here?

Reply:
Yeah. Ditto

Reply:
I shall remove the coupling